pytest_splunk_addon/standard_lib/fields_tests/test_templates.py

```diff
@@ -199,22 +203,23 @@ def test_requirements_fields(
             if value != fields_from_splunk.get(field):
                 wrong_value_fields[field] = fields_from_splunk.get(field)
 
-        failure_message = ""
-
-        for field, value in wrong_value_fields.items():
-            failure_message += (
-                f"Field {field} has value {value} and should has {fields[field]}\n"
-            )
-
-        if failure_message:
-            self.logger.error(f"Fields with wrong values: {failure_message}")
+        exc_message = get_table_output(
+            headers=["Field", "Splunk value", "Expected value"],
+            value_list=[
+                [
+                    str(field),
+                    str(value),
+                    str(fields[field]),
+                ]
+                for field, value in wrong_value_fields.items()
+            ],
+        )
 
-        assert (
-            missing_fields == []
-        ), f"Not all required fields found in Splunk. Missing fields: {', '.join(missing_fields)}"
-        assert (
-            wrong_value_fields == {}
-        ), f"Not all required fields have correct values in Splunk. Wrong field values: {failure_message} Search string: {search}"
+        self.logger.error(exc_message)
```

Comment:
what is your goal here? to log error? if so log only when `wrong_value_fields` is not empty + add some header why you print this error
if you like to always log it, just change to info